TACC/Core-CMS-Resources
# chore(texascale): fp-1903 sync tup-ui login component changes to core-styles

## Overview

Updates based on styles changed from https://github.com/TACC/Core-Styles/pull/41.

## Related

- [FP-1903](https://jira.tacc.utexas.edu/browse/FP-1903)
- required by https://github.com/TACC/Core-CMS/pull/531

## Changes

- load element styles from core-styles at new file name

## Testing

1. Open Core-CMS at branch of [TACC/Core-CMS#531](https://github.com/TACC/Core-CMS/pull/531).
2. `npm ci`
3. `npm run build --project=texascale-org`
4. Verify no import/build error.

## UI

Skipped.

_Texascale is difficult to test, and not a candidate for new deploys, until [CEPWMA-680](https://jira.tacc.utexas.edu/browse/CEPWMA-680)._
